### PR TITLE
Set up client and server's Redis instances

### PR DESCRIPTION
**Pull request by lotif**
# PR Type
Feature

# Short Description

Clickup Ticket(s): https://app.clickup.com/t/8687c8dhz

Adding instructions to start redis and poetry dependency. Tested with the following commands in python shell:
```shell
(venv) marcelolotif@VI-FM46N1XVYX FLorist % python
Python 3.9.6 (default, Nov 10 2023, 13:38:27) 
[Clang 15.0.0 (clang-1500.1.0.2.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import redis
>>> server_conn = redis.Redis(host="localhost", port=6379)
>>> test_server_dict = {"test":"server"}
>>> server_conn.hset("server_dict", mapping=test_server_dict)
1
>>> server_conn.hgetall("server_dict")
{b'test': b'server'}
>>> client_conn = redis.Redis(host="localhost", port=6380)
>>> test_client_dict = {"test":"client"}
>>> client_conn.hset("client_dict", mapping=test_client_dict)
1
>>> client_conn.hgetall("client_dict")
{b'test': b'client'}
```

# Tests Added
NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +35,32 @@ Then install the project dependencies:
 yarn
 ```
 
-### Running the server
+### Pulling Redis' Docker
+
+Redis is used to facilitate the communication between FL4Health/Flower and
```
Comment (jewelltaylor) [inline README.md:40]: maybe worth adding hyperlink to redis or redis container for context

Comment (jewelltaylor) [reply README.md:40]: what do you mean by communication between FL4Health/Flower and FLorist's servers and clients? FLorist server and clients are using FL4Health/Flower under the hood so I think thats where I am getting confused. 

I was thinking redis would be a way for our FLorist Manager to fetch reported metrics from running servers and clients during runs. 

Not necessarily asking you to change the documentation but I would appreciate if you could clear up my misunderstanding so I can better understand the role of redis

Comment (lotif) [reply README.md:40]: > redis would be a way for our FLorist Manager to fetch reported metrics from running servers and clients during runs.

Yes, that's what I meant, I just chose words poorly :) Looks like your understanding is correct.

Comment (lotif) [reply README.md]: @jewelltaylor fixed

**Pull request merged**